src/main/java/nextstep/ladder/ladder/Line.java

```diff
@@ -0,0 +1,42 @@
+package nextstep.ladder.ladder;
+
+import nextstep.ladder.strategy.LadderStrategy;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Line {
+
+    private static final int START_INDEX = 0;
+
+    private final List<Boolean> points;
+
+    private Line(List<Boolean> points) {
+        this.points = points;
+    }
+
+    public static Line of(int countOfPerson, LadderStrategy strategy) {
+        List<Boolean> pointList = new ArrayList<>();
+        IntStream.range(START_INDEX, countOfPerson)
+                .forEach(index -> addPoint(pointList, strategy));
+        return new Line(pointList);
+    }
+
+    public List<Boolean> getPoints() {
+        return Collections.unmodifiableList(points);
+    }
+
+    private static void addPoint(List<Boolean> pointList, LadderStrategy strategy) {
+        boolean previousPoint = false;
+        if (!pointList.isEmpty()) {
+            previousPoint = pointList.get(pointList.size() - 1);
+        }
+        if (previousPoint) {
+            pointList.add(false);
+            return;
+        }
+        pointList.add(strategy.creatable());
+    }
```

Comment:
```suggestion

    private static void addPoint(List<Boolean> pointList, LadderStrategy strategy) {
        boolean previousPoint = false;
        if (!pointList.isEmpty()) {
            previousPoint = pointList.get(pointList.size() - 1);
        }
        if (previousPoint) {
            pointList.add(false);
            return;
        }
        pointList.add(strategy.creatable());
    }
    
    public List<Boolean> getPoints() {
        return Collections.unmodifiableList(points);
    }
```

클래스 구현 순서는 아래와 같습니다 :)

```
class A {
    상수(static final) 또는 클래스 변수
    인스턴스 변수
    생성자
    팩토리 메소드
    메소드
    기본 메소드 (equals, hashCode, toString)
}
```

Reply:
접근제어자가 private 면 항상 가장 아래에 위치해야 한다고 생각했었습니다.
다음 미션에서는 피드백 해주신 순서를 지키도록 해보겠습니다.